static/js/click_handlers.js

```diff
@@ -447,11 +447,22 @@ export function initialize() {
 
     $("body").on("click", "#recent_topics_table .on_hover_topic_read", (e) => {
         e.stopPropagation();
-        const topic_row_index = $(e.target).closest("tr").index();
+        const $elt = $(e.currentTarget);
+        const topic_row_index = $elt.closest("tr").index();
         recent_topics_ui.focus_clicked_element(topic_row_index, recent_topics_ui.COLUMNS.read);
-        const stream_id = Number.parseInt($(e.currentTarget).attr("data-stream-id"), 10);
-        const topic = $(e.currentTarget).attr("data-topic-name");
-        unread_ops.mark_topic_as_read(stream_id, topic);
+        if (e.currentTarget.classList.contains("unread_count_pm")) {
```

Comment:
This check might be cleaner as just checking if `$elt` has a `data-user-ids-string` attribute, since then we're checking the same value we use 2 lines down.